tdrs-backend/tdpservice/users/models.py

```diff
@@ -14,6 +14,9 @@ class User(AbstractUser):
     stt = models.ForeignKey(STT, on_delete=models.CASCADE, blank=True, null=True)
     region = models.ForeignKey(Region, on_delete=models.CASCADE, blank=True, null=True)
 
+    # The unique `sub` UUID from decoded login.gov payloads.
+    sub = models.UUIDField(editable=False, blank=True, null=True, unique=True)
```

Comment:
As noted above, this will addition will generate a corresponding migration file to apply to the DB.